libc/test/src/math/smoke/CMakeLists.txt

```diff
@@ -151,8 +151,32 @@ add_fp_unittest(
     AddTest.h
   DEPENDS
     libc.src.math.fadd
-    libc.src.__support.FPUtil.basic_operations
+)
+
+add_fp_unittest(
+  faddl_test
+  NEED_MPFR
+  SUITE
+    libc-math-unittests
+  SRCS
+    faddl_test.cpp
+  HDRS
+    AddTest.h
+  DEPENDS
+    libc.src.math.faddl
+)
```

Comment:
Should add `libc.hdr.*` on all tests that include smoke/AddTest.h, including tests for `fadd*`.